pkg/virt-controller/services/template.go

```diff
@@ -60,12 +61,52 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VirtualMachine) (*kubev1.P
 	var volumes []kubev1.Volume
 	var userId int64 = 0
 	var privileged bool = true
+	var volumesMounts []kubev1.VolumeMount
 
 	gracePeriodSeconds := v1.DefaultGracePeriodSeconds
 	if vm.Spec.TerminationGracePeriodSeconds != nil {
 		gracePeriodSeconds = *vm.Spec.TerminationGracePeriodSeconds
 	}
 
+	volumesMounts = append(volumesMounts, kubev1.VolumeMount{
+		Name:      "virt-share-dir",
+		MountPath: t.virtShareDir,
+	})
+	volumesMounts = append(volumesMounts, kubev1.VolumeMount{
+		Name:      "libvirt-runtime",
+		MountPath: "/var/run/libvirt",
+	})
+	volumesMounts = append(volumesMounts, kubev1.VolumeMount{
+		Name:      "host-dev",
+		MountPath: "/host-dev",
+	})
+	volumesMounts = append(volumesMounts, kubev1.VolumeMount{
+		Name:      "host-sys",
+		MountPath: "/host-sys",
+	})
+	for _, volume := range vm.Spec.Volumes {
+		if volume.PersistentVolumeClaim != nil {
+			volumesMounts = append(volumesMounts, kubev1.VolumeMount{
+				Name: volume.Name,
+				MountPath: filepath.Join(
+					"/var/run/kubevirt-private",
+					namespace,
```

Comment:
you don't have to put the namespace and domain in the path anymore for the private directory. It's all self contained in the container now and not visible to any other pods.

It's also fine to leave this the way it is. I just wanted to point that bit out.